service/controller/aws/cluster.go

```diff
@@ -47,12 +47,12 @@ type ClusterConfig struct {
 	ResourceNamespace  string
 }
 
-type Cluster struct {
+type LegacyCluster struct {
```

Comment:
I think file should be `legacy_cluster.go`?

Reply:
Fine. Done.